dtmi/impinj/fixedreader/r700-13.json

```diff
@@ -0,0 +1,3882 @@
+{
+  "@id": "dtmi:impinj:R700;13",
```

Comment:
based on the file path, seems this id is missing the `fixedreader` segment in the dtmi.

Reply:
updated!  moved r700-13.json to same folder /dtmi/impinj